src/material/button/button-lazy-loader.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {DOCUMENT} from '@angular/common';
+import {
+  ANIMATION_MODULE_TYPE,
+  ElementRef,
+  Injectable,
+  NgZone,
+  OnDestroy,
+  inject,
+} from '@angular/core';
+import {
+  MAT_RIPPLE_GLOBAL_OPTIONS,
+  MatRipple,
+  RippleConfig,
+  RippleGlobalOptions,
+  RippleRenderer,
+  RippleTarget,
+} from '@angular/material/core';
+import {Platform} from '@angular/cdk/platform';
+
+/** The options for the MatButtonRippleLoader's event listeners. */
+const OPTIONS = {capture: true};
+
+/** The events that should trigger the initialization of the ripple. */
+const RIPPLE_INTERACTION_EVENTS = ['focus', 'click', 'mouseenter', 'touchstart'];
```

Comment:
optional / personal preference: I tend to prefer using camelCase for local (non-exported), non-primitive values like this